src/webcomponents/job/opencga-job-detail-log.js

```diff
@@ -131,39 +131,44 @@ export default class OpencgaJobDetailLog extends LitElement {
         const command = params.command || this._config.command;
         // const offset = params.offset || 0;
         // console.log("request ", "command", command, "params", params, "offset", offset, "append", append);
-
-        this.opencgaSession.opencgaClient.jobs()[command + "Log"](job.id, {
-            study: this.opencgaSession.study.fqn,
-            lines: this._config.lines,
-            type: this._config.type,
-            offset: params.offset || 0,
-            ...params
-        }).then(restResponse => {
-            const result = restResponse.getResult(0);
-            if (result.content) {
+        if (["PENDING", "ABORTED", "RUNNING"].every(status => status !== job?.internal?.status.id)) {
```

Comment:
I think in this case `every` is not the most appropriate method. I would suggest using `includes` instead (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes):

```
const statusWithoutLogs = ["PENDING", "ABORTED", "RUNNING"];
if (!statusWithoutLogs.includes(join?.internal?.status?.id?.toUpperCase())) {
    ....
}
```

Also remember to convert the status Id to upper case to prevent issues.

Reply:
Done!